### PR TITLE
Xcode fixes for macOS support

### PR DESCRIPTION
**Pull request by Andy608**
Added in Duncan's files to the Xcode project so the project builds correctly on Mac.

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Did you add them to the libopenrct2 target only?

**Comment by Andy608** [comment]
I don't notice any duplicates. I only added the files to libopenrct2 in their respective directories.

File: contributors.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ The following people are not part of the development team, but have been contrib
 * Olivier Wervers (oli414) - Remove unused objects command, various bugfixes
 * Christian Schubert (Osmodium) - Ensuring custom user content folders, incl. open folder.
 * (Xkeeper0) - Improved banner tooltips; multiplayer status in toolbar.
+* Andrew Rimpici (Andy608) - Xcode fixes for macOS support.
```
Comment (AaronVanGeffen) [inline contributors.md]: There's a 'toolchain' section below. I think it fits your contribution better.

Comment (Andy608) [reply contributors.md]: Thanks, I just updated the file. I wasn't sure where to put it.

**Pull request merged**